client/src/views/Knowledge/DocsClusters/DocsClusters.vue

```diff
@@ -1,16 +1,19 @@
 <template>
   <div class="view-container">
-    <div class="d-flex flex-column h-100">
-      <div class="search-row">
-        <search-bar :placeholder="`Search for documents including a specific keyword (e.g. IL-6)...`" />
-      </div>
-      <settings-bar>
-        <div slot="right">
-          <settings />
+    <main>
+      <div class="d-flex flex-column h-100">
+        <div class="search-row">
+          <search-bar :placeholder="`Search for documents including a specific keyword (e.g. IL-6)...`" />
         </div>
-      </settings-bar>
-      <grafer class="grafer" layer="epi" />
-    </div>
+        <settings-bar>
+          <counters slot="left" :data="[{ name: 'Documents', value: 176000 }]"/>
```

Comment:
Can't we get this count from the data itself?

Reply:
I need to connect this all to bgraph to allow for search. This part is only for static rendering the next step would be to connect this with bgraph and that'll give us a live counter.